cscs-checks/mch/alltoallv.py

```diff
@@ -0,0 +1,92 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+@rfm.parameterized_test([''], ['--nocomm'], ['--nocomp'])
```

Comment:
I know it's handy, but I wouldn't pass the option here. I would rename the `exec_parameter` to `variant` and pass three distinct values: `default`, `nocomm`, `nocomp`. I would then set up the executable options accordingly. The way it is now will produce ugly test names, e.g., `Alltoallv___nocomm`.